senior_teacher.rb

```diff
@@ -26,15 +19,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
     response = ""
     if rating > 90
```

Comment:
Forgot one thing.  You could pull up set_performance_rating and reference a constant in each Teacher class... maybe HIGH_RATING that would dry it up even more.